scss/_variables.scss

```diff
@@ -842,7 +881,7 @@ $table-striped-columns-order:           even !default;
 $table-group-separator-color:           currentcolor !default;
 
```

Comment:
**[Out?]** Should we revert `$table-bg: transparent` ? Can't remember why we've chosen a defined background.